modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -31,7 +31,7 @@ public class KafkaContainer extends GenericContainer<KafkaContainer> {
 
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("confluentinc/cp-kafka");
 
-    private static final String DEFAULT_TAG = "5.4.3";
+    private static final String DEFAULT_TAG = "7.5.3";
```

Comment:
Default constructors are deprecated and if it is used then this change could break some tests.

Reply:
@eddumelendez I can revert the DEFAULT_TAG change, the idea of PR was to simply update the docs to point a newer version of cp-kafka image to 7.5.3, instead of a very old image like `6.2.1` in `constructorWithVersion` of  `KafkaContainerTest.java`

Reply:
Thanks for the offer. But, currently we are not doing so for any module. Unless, new versions are not working with the Testcontainers implementation.

Reply:
The versions testcontainers is pointing to are unsupported and have a number of cves - is it really intended to work that way?

Reply:
The way it is supposed to work is users providing the image and tag they want to use and not relying on deprecated default constructors. 

Updating docs is fine 👍

Reply:
@kiview iiuc the docs are generated based on the code present in test file
example - section containing `// withExternalZookeeper {` in https://github.com/alok123t/testcontainers-java/blob/666ec53e87ec8821ed64d93256ed012849aa28b4/modules/kafka/src/test/java/org/testcontainers/containers/KafkaContainerTest.java#L93..L97 is used to update https://java.testcontainers.org/modules/kafka/

Is it fine to update the above test file to reflect this then? If so, I can update the PR accordingly